Megatron-LM/pretrain_gpt2.py

```diff
@@ -110,10 +111,16 @@ def get_optimizer(model, args):
             if not hasattr(param, 'model_parallel'):
                 param.model_parallel = False
 
-    # Use Adam.
-    optimizer = Adam(param_groups,
-                     lr=args.lr, weight_decay=args.weight_decay)
+    if args.cpu_optimizer:
+        cpu_adam_optimizer = torch.optim.Adam if args.cpu_torch_adam else DeepSpeedCPUAdam
```

Comment:
Can we move the DeepSpeedCPUAdam import into this so we don't import unless needed